crates/core/src/operations/subscribe.rs

```diff
@@ -66,13 +66,18 @@ pub(crate) async fn request_subscribe(
     sub_op: SubscribeOp,
 ) -> Result<(), OpError> {
     if let Some(SubscribeState::PrepareRequest { id, key }) = &sub_op.state {
-        // Find a remote peer to handle the subscription
-        const EMPTY: &[PeerId] = &[];
-        let target = match op_manager.ring.closest_potentially_caching(key, EMPTY) {
+        // Phase 1 fix for issue #1848: Properly find next-best peer for subscription
```

Comment:
Remove this explicit comment: "// Phase 1 fix for issue #1848: Properly find next-best peer for subscription"